packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -211,6 +211,7 @@ export interface ISummaryBaseConfiguration {
     initialSummarizerDelayMs: number;
 
     /**
+     * @deprecated
```

Comment:
Why not deprecate this is `main` and then nuke in this PR?

Reply:
I did not know I could  :)  If so, I can, for sure.

Reply:
See [Communicating breaking changes](https://github.com/microsoft/FluidFramework/wiki/Communicating-breaking-changes#breaking-changes-vs-upcoming-changes)
> An important note to remember is that we may announce upcoming changes during a **minor or major** release.